yt/yt/server/lib/chunk_pools/ordered_chunk_pool.cpp

```diff
@@ -404,26 +396,147 @@ class TOrderedChunkPool
             : JobSizeConstraints_->GetDataWeightPerJob();
     }
 
-    IChunkPoolOutput::TCookie AddPrimaryDataSlice(
+    i64 GetCorrectedCurrentJobDataWeight()
```

Comment:
Maybe just GetCurrentJobDataWeight to avoid adding new term in already term-overloaded code?